data/sql/updates/pending_db_world/rev_1643395587559675400.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1643395587559675400');
+
+CREATE TABLE IF NOT EXISTS `instance_saved_data` (
+  `instance_id` int unsigned NOT NULL,
+  `gameobject_entry` int unsigned NOT NULL,
+  `gameobject_guid` int unsigned NOT NULL,
+  `gameobject_state` int unsigned DEFAULT '0',
+  PRIMARY KEY (`instance_id`,`gameobject_entry`,`gameobject_guid`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3 COLLATE=utf8_bin;
```

Comment:
https://www.azerothcore.org/wiki/sql-standards#tables-and-columns

use utf8mb4, also it is good if you add DROP TABLE IF EXISTS just to follow the way tables are already created